gef-ui/server/controllers/unissued-facilities/validation.js

```diff
@@ -220,7 +222,10 @@ const facilityValidation = async (body, query, params) => {
   }
 
   if (coverStartDateIsFullyComplete) {
-    coverStartDate = set(new Date(), { year: coverStartDateYear, month: coverStartDateMonth - 1, date: coverStartDateDay });
+    const value = shouldCoverStartOnSubmission
```

Comment:
can we use an `if/else` statement for this? the ternary operator gets harder to read if it's too complex

Reply:
Have updated.